src/App.js

```diff
@@ -1,14 +1,21 @@
-import './App.css';
-import Calculator from './components/calculator';
-import ApiCall from './components/apiCall';
+import { Routes, Route } from 'react-router-dom';
 
-function App() {
-  return (
-    <div className="calculator-holder">
-      <Calculator />
-      <ApiCall />
-    </div>
-  );
-}
+import Layout from './components/Layout';
+import Home from './routes/Home';
+import CalculatorPage from './routes/CalculatorPage';
+import Quote from './routes/Quote';
+import NotMatch from './routes/NotMatch';
+import './styles/App.css';
+
+const App = () => (
+  <Routes>
+    <Route path="/" element={<Layout />}>
+      <Route index element={<Home />} />
```

Comment:
- [optional] to make it more unique, you can highlight the navlinks so they indicate which page a user is currently in. great job. 👏🏾